google-cloud-compute-v1/acceptance/google/cloud/compute/v1/test_use_compute_pagination.rb

```diff
@@ -0,0 +1,41 @@
+require 'securerandom'
+require "minitest/autorun"
+
+require "google/cloud/compute/v1/zones"
+require "google/cloud/compute/v1/regions"
+
+$default_zone = 'us-central1-a'
+$default_project = ENV['PROJECT_ID']
```

Comment:
Style: Try to avoid using globals. (They can interfere with other tests, or even with code.) For this sort of thing, use constants or memoized accessors in the test class below, or even just set instance variables in the `setup` method.

Reply:
moved to setup()